fusor-ember-cli/app/templates/components/cancel-back-next.hbs

```diff
@@ -1,18 +1,19 @@
-<div class="col-md-9 rhci-steps-buttons">
+<div class="col-xs-8 col-sm-8 col-md-9 col-lg-10 col-xs-push-4 col-sm-push-4 col-md-push-3 col-lg-push-2 sidebar-pf-right rhci-steps-buttons cancel-back-next">
   {{#if isNew}}
-      <button class="btn btn-default" {{action 'cancelAndRollbackNewDeployment'}}>
+      <button class="btn btn-default cancel-button" {{action 'cancelAndRollbackNewDeployment'}}>
         Cancel
       </button>
    {{else}}
-      <button class="btn btn-default" {{action 'openCancelDeploymentModal'}} disabled={{disableCancel}}>
+      <button class="btn btn-default cancel-button" {{action 'openCancelDeploymentModal'}} disabled={{disableCancel}}>
         Cancel
       </button>
   {{/if}}
   {{#link-to backRouteName disabled=disableBack role="button" class='btn btn-default'}}
       <i class="fa fa-angle-left"></i> Back
   {{/link-to}}
+  &nbsp;
```

Comment:
@cfchase, @eriknelson, I have no idea why, but adding this `&nbsp'` helped eliminate the extra padding caused if `{{#cancel-back-next}}` was in block form. The HTML mark up appeared to be the same???